pkg/git/github/github.go

```diff
@@ -50,14 +49,15 @@ func newGitHubClient(ctx context.Context, endpoint, token string) (*github.Clien
 	return client, nil
 }
 
-func (g *GitHub) DownloadContents(org, repo, path string) (string, error) {
+func (g *GitHub) DownloadContents(org, repo, path, branch string) (string, error) {
 	ctx := context.Background()
 	client, err := newGitHubClient(ctx, g.Endpoint, g.Token)
 	if err != nil {
 		return "", err
 	}
 
-	r, err := client.Repositories.DownloadContents(ctx, org, repo, path, nil)
+	opt := github.RepositoryContentGetOptions{Ref: branch}
```

Comment:
Any reason not to use `&github.RepositoryContentGetoptions` and pass in the variable instead of the reference?

Reply:
The library that we're using requires that `RepositoryContentGetOptions` type to be passed in. Could prob rename some of the Github stuff in there just to make it less confusing as to what is ours and what is the lib.